### ISSUE TITLE
Editorial: Re-write

### ISSUE THREAD

**Issue by ghost** [lnbook/lnbook#679]
**Which part needs editing?**
05_node_operations.asciidoc
Line 618

**What editing does this need?**
This needs to be rewritten - it isn't clear.

**How does it currently read?**
A node operator can initiate a submarine swap and send all available channel balances to the other party who will send them as a result on-chain bitcoin in exchange.

**How should it read?**
I don't fully understand the underlying technology, but here is my best guess:

_A node operator can initiate a submarine swap and send all available channel balances to another party who will, in turn, send them on-chain as bitcoin._

**Comment by ckapilla** [comment]
I have to disagree that this is clearer -- it sounds like "on-chain" is something that gets sent, and that doesn't really make sense. Here is another try: "A node operator can initiate a submarine swap by sending all available channel balances to another party. The receiver will then send on-chain bitcoin of equal value to the originator of the swap."

**Comment by ghost** [comment]
@ckapilla  Yes, that's much clearer.  I was just having problems with "who will send them as a result on-chain bitcoin in exchange" which makes no sense at all.
